config-editor/config-editor-ui/src/app/services/store/config-store.service.ts

```diff
@@ -225,12 +232,17 @@ export class ConfigStoreService {
   }
 
   reloadStoreAndRelease(): Observable<any> {
-    const testCaseMapFun = this.metaDataMap.testing.testCaseEnabled ? this.configLoaderService.getTestCases() : of({});
     return forkJoin(
       this.configLoaderService.getConfigs(),
-      this.configLoaderService.getRelease(),
-      testCaseMapFun
-    ).pipe(map(([configs, release, testCaseMap]) => {
+      this.configLoaderService.getRelease()
+    ).pipe(mergeMap(([configs, release]) => {
+      return forkJoin(
+        of(configs),
+        of(release),
+        this.store.getValue().testSpecificationTesters.test_case_testing.length > 0 ? this.configLoaderService.getTestCases() : of({})
```

Comment:
you can revert back to previous simpler version, and move this line out of the observable operations